pset/blinder.go

```diff
@@ -113,6 +136,178 @@ func (b *blinder) BlindOutputs() error {
 	return nil
 }
 
+//BlindOutputs method blinds pset's issuance pseudo input
+func (b *blinder) blindInputs() error {
+	for index, input := range b.pset.UnsignedTx.Inputs {
+		if input.Issuance != nil {
+			if b.issuanceBlindingPrivateKeys == nil {
+				return nil
+			}
```

Comment:
this could be done before iterating over the inputs

Reply:
Done